### ISSUE TITLE
Move example files to an example folder

### ISSUE THREAD

**Issue by ElSnoMan** [ElSnoMan/pyclinic#36]
`test_account_service.py` and `account_service.py` and their corresponding `models` folder are just examples. We should move them into a "example" folder so that's clear.

**Comment by gleekzorp** [comment]
I get the following error when running `poetry run poe test`
```
tests/test_account_service.py:4: in <module>
    from tests import account_service
tests/account_service.py:5: in <module>
    from models.account_model import Nate, Token, User
E   ModuleNotFoundError: No module named 'models'
```

Steps I used to get the project to my machine

- Forked
- Cloned my fork to my mac
- `poetry install`
- `poetry run poe test`
  - Ran into the error

It seems this models directory is being ignored in your `.gitignore`.  If I comment out all of `account_service.py` and `test_account_service.py` and run `poetry run poe test` everything works.

**Comment by gleekzorp** [comment]
Looks like this was fixed with pull request #40
